### PR TITLE
refactor(Checkbox): Refactored checkbox to ts

### PR DESCRIPTION
**Pull request by wichniowski**
This PR will refactor Checkbox, RadioButton and all controlled Input Field Components to TS.

### CODE REVIEW

File: packages/forma-36-react-components/src/components/RadioButton/RadioButton.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
-import React from 'react';
+import React, { SFC } from 'react';
 import ControlledInput, {
   ControlledInputDefaultProps,
   ControlledInputPropTypes,
 } from '../ControlledInput/ControlledInput';
 
-const RadioButton = props => <ControlledInput {...props} type="radio" />;
+const RadioButton: SFC<ControlledInputPropTypes> = props => (
```
Comment (suevalov) [inline packages/forma-36-react-components/src/components/RadioButton/RadioButton.tsx]: I believe that using `SFC` is deprecated. There should `FunctionalComponent` type or something like this.

File: packages/forma-36-react-components/src/components/RadioButtonField/RadioButtonField.tsx [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
+import React, { Component } from 'react';
 import ControlledInputField, {
   ControlledInputFieldDefaultProps,
   ControlledInputFieldPropTypes,
 } from '../ControlledInputField/ControlledInputField';
 
-class RadioButtonField extends React.Component {
-  static propTypes = ControlledInputFieldPropTypes;
-
+class RadioButtonField extends Component<ControlledInputFieldPropTypes> {
```
Comment (suevalov) [inline packages/forma-36-react-components/src/components/RadioButtonField/RadioButtonField.tsx]: export

File: packages/forma-36-react-components/src/components/RadioButtonField/RadioButtonField.tsx [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
   static defaultProps = {
```
Comment (suevalov) [inline packages/forma-36-react-components/src/components/RadioButtonField/RadioButtonField.tsx]: partial type

File: packages/forma-36-react-components/src/components/ControlledInput/ControlledInput.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+const styles = './ControlledInput.css';
```
Comment (suevalov) [inline packages/forma-36-react-components/src/components/ControlledInput/ControlledInput.tsx]: const styles = require('./ControlledInput.css')

File: packages/forma-36-react-components/src/components/ControlledInput/__snapshots__/ControlledInput.test.tsx.snap [hunk 5]
```diff
@@ -17,11 +19,13 @@ exports[`renders the component with a value 1`] = `
 exports[`renders the component with all required props 1`] = `
 <input
   aria-label="ControlledInput"
-  className="ControlledInput"
+  className=""
```
Comment (suevalov) [inline packages/forma-36-react-components/src/components/ControlledInput/__snapshots__/ControlledInput.test.tsx.snap]: Changes in this file are caused by the error above.

Comment (wichniowski) [reply packages/forma-36-react-components/src/components/ControlledInput/__snapshots__/ControlledInput.test.tsx.snap]: oh dammit!

Comment (wichniowski) [reply packages/forma-36-react-components/src/components/ControlledInput/__snapshots__/ControlledInput.test.tsx.snap]: ok fixed thanks for the heads up!

**Pull request merged**